images/iscsi-target-tgtd/run-tgt.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# https://fedoraproject.org/wiki/Scsi-target-utils_Quickstart_Guide
+
+set -e
+
+die() { echo "ERR: $@" ; exit 2 ; }
+
+SIZE_MB=${SIZE:-1024}
+WWN=iqn.2017-01.io.kubevirt:sn.42
+
+if [[ "$GENERATE_DEMO_OS_SEED" == "alpine" ]]; then
```

Comment:
Maybe instead of having three different branches on how to start this pod, we could just have one way, where it provides all three files at once?